ttnn/cpp/ttnn/device_operation.hpp

```diff
@@ -89,9 +89,9 @@ concept DeviceOperationWithCustomProgramCacheConcept = DeviceOperationConcept<de
 };
 
 template <typename... Ts>
-[[nodiscard]] std::variant<Ts...> constexpr map_index_to_variant(std::size_t i, std::variant<Ts...>) {
+[[nodiscard]] std::variant<Ts...> map_index_to_variant(std::size_t i, std::variant<Ts...>) {
```

Comment:
@arakhmati ptal

Reply:
Any comment from @arakhmati ? 🙏

Reply:
He got sick, should be back tomorrow.

Reply:
Looks okay to me. I believe the compiler should do the same thing whether it's contstexpr or not